packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_sidecar/disk_usage.py

```diff
@@ -12,11 +12,15 @@
 
 @log_decorator(_logger, level=logging.DEBUG)
 async def update_disk_usage(
-    rabbitmq_rpc_client: RabbitMQRPCClient, *, usage: dict[str, DiskUsage]
+    rabbitmq_rpc_client: RabbitMQRPCClient,
+    *,
+    node_id: NodeID,
+    usage: dict[str, DiskUsage],
 ) -> None:
+    rpc_namespace = RPCNamespace.from_entries(
```

Comment:
This reminds me a bit the cardinality issue we faced with prometheous. How does this scale, i.e. are these namespaces deallocated cleanly when not necessary anymore? Who is responsible of that?

Reply:
I have replied above about that. In short: it's not a problem for our use case.

I have tested your concern about cleanup and rabbitmq removes all queues when this disconnects. No issues there.